<ui version="4.0" >
 <class>PhotogrammetryBase</class>
 <widget class="QMainWindow" name="PhotogrammetryBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QSplitter" name="splitter_3" >
    <property name="geometry" >
     <rect>
      <x>9</x>
      <y>9</y>
      <width>783</width>
      <height>689</height>
     </rect>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="tabShape" >
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabInput" >
      <attribute name="title" >
       <string>源数据</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5" >
       <item>
        <widget class="QSplitter" name="splitter" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="" >
          <layout class="QVBoxLayout" name="verticalLayout_4" >
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3" >
             <item>
              <widget class="QLabel" name="photoPointsLabel" >
               <property name="text" >
                <string>相片点:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTableView" name="photoPointsView" />
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="fiducialMarkGroupBox" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title" >
              <string>框标距:</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2" >
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout" >
                <item>
                 <widget class="QLabel" name="label" >
                  <property name="text" >
                   <string>理论值:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="tfxLabel" >
                  <property name="text" >
                   <string>(X):</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="tfxSpinBox" >
                  <property name="minimum" >
                   <double>0.000000000000000</double>
                  </property>
                  <property name="maximum" >
                   <double>1000.000000000000000</double>
                  </property>
                  <property name="value" >
                   <double>0.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="tfyLabel" >
                  <property name="text" >
                   <string>(Y):</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="tfySpinBox" >
                  <property name="minimum" >
                   <double>0.000000000000000</double>
                  </property>
                  <property name="maximum" >
                   <double>1000.000000000000000</double>
                  </property>
                  <property name="value" >
                   <double>0.000000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2" >
                <item>
                 <widget class="QLabel" name="label_2" >
                  <property name="text" >
                   <string>实际值:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="rfxLabel" >
                  <property name="text" >
                   <string>(X):</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="rfxSpinBox" >
                  <property name="minimum" >
                   <double>0.000000000000000</double>
                  </property>
                  <property name="maximum" >
                   <double>1000.000000000000000</double>
                  </property>
                  <property name="value" >
                   <double>0.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="rfyLabel" >
                  <property name="text" >
                   <string>(Y):</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="rfySpinBox" >
                  <property name="maximum" >
                   <double>1000.000000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
             <zorder>rfxSpinBox</zorder>
             <zorder>rfyLabel</zorder>
             <zorder>tfxSpinBox</zorder>
             <zorder>tfyLabel</zorder>
             <zorder>rfySpinBox</zorder>
             <zorder>tfySpinBox</zorder>
             <zorder>rfxLabel</zorder>
             <zorder>tfxLabel</zorder>
             <zorder>label</zorder>
             <zorder>label_2</zorder>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="" >
          <layout class="QVBoxLayout" name="verticalLayout_11" >
           <item>
            <widget class="QLabel" name="controlPointsLabel" >
             <property name="text" >
              <string>控制点:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableView" name="controlPointsView" />
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu" >
    <property name="title" >
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="fileOpenAction" />
    <addaction name="fileNewAction" />
    <addaction name="fileSaveAction" />
    <addaction name="separator" />
    <addaction name="fileExitAction" />
   </widget>
   <widget class="QMenu" name="orientationMenu" >
    <property name="title" >
     <string>定向(O)</string>
    </property>
    <addaction name="orientationRelativeAction" />
    <addaction name="orientationAbsoluteAction" />
   </widget>
   <widget class="QMenu" name="intersectionMenu" >
    <property name="title" >
     <string>交会(&amp;I)</string>
    </property>
    <addaction name="intersectionForwardAction" />
    <addaction name="intersectionBackwardAction" />
   </widget>
   <widget class="QMenu" name="helpMenu" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="title" >
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="helpAboutAction" />
   </widget>
   <widget class="QMenu" name="menuOneStep_O" >
    <property name="title" >
     <string>光束法(&amp;C)</string>
    </property>
    <addaction name="onestepAction" />
   </widget>
   <addaction name="fileMenu" />
   <addaction name="orientationMenu" />
   <addaction name="intersectionMenu" />
   <addaction name="menuOneStep_O" />
   <addaction name="helpMenu" />
  </widget>
  <action name="fileOpenAction" >
   <property name="text" >
    <string>打开(&amp;O)</string>
   </property>
  </action>
  <action name="fileNewAction" >
   <property name="text" >
    <string>新建(&amp;N)</string>
   </property>
  </action>
  <action name="fileSaveAction" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>保存(&amp;S)</string>
   </property>
  </action>
  <action name="fileExitAction" >
   <property name="text" >
    <string>退出(X)</string>
   </property>
  </action>
  <action name="orientationRelativeAction" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>相对定向(&amp;R)</string>
   </property>
  </action>
  <action name="orientationAbsoluteAction" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>绝对定向(&amp;A)</string>
   </property>
  </action>
  <action name="intersectionForwardAction" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>前方交会(&amp;F)</string>
   </property>
  </action>
  <action name="intersectionBackwardAction" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>后方交会(&amp;B)</string>
   </property>
  </action>
  <action name="helpAboutAction" >
   <property name="text" >
    <string>关于(&amp;A)</string>
   </property>
  </action>
  <action name="onestepAction" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>一步法(&amp;O)</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>photoPointsView</tabstop>
  <tabstop>controlPointsView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
